samples/perf-virtualgridlist-native/src/App.js

```diff
@@ -8,13 +8,19 @@ import css from './App.module.less';
 
 const items = [];
 
+const svgGenerator = (width, height, bgColor, textColor, customText) => (
+	`data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 ${width} ${height}' width='${width}' height='${height}'%3E` +
+	`%3Crect width='${width}' height='${height}' fill='%23${bgColor}'%3E%3C/rect%3E` +
+	`%3Ctext x='50%25' y='50%25' dominant-baseline='middle' text-anchor='middle' font-size='36px' fill='%23${textColor}'%3E${customText}%3C/text%3E%3C/svg%3E`
+);
+
 for (let i = 0; i < 1000; i++) {
 	let color = Math.floor((Math.random() * (0x1000000 - 0x101010)) + 0x101010).toString(16),
 		count = ('00' + i).slice(-3);
 	items.push({
 		text: 'Item ' + count,
 		subText: 'SubItem ' + count,
-		url: 'http://via.placeholder.com/193x150/' + color + '/ffffff/png?text=Image ' + i
+		url: svgGenerator(300, 300, color, 'ffffff', `Image ${i}`)
```

Comment:
```suggestion
		url: svgGenerator(193, 150, color, 'ffffff', `Image ${i}`)
```
And also please fix perf-virtualgridlist-translate